src/deep-cache/lib/SharedCache.js

```diff
@@ -70,11 +71,61 @@ export class SharedCache {
    * @returns {String}
    */
   buildKeyFromLambdaRuntime(runtime) {
-    let payload = JSON.stringify(runtime.request.data);
+    return runtime.context && runtime.context.has('invokedFunctionArn') ?
+      `${runtime.context.getOption('invokedFunctionArn')}#${SharedCache._payloadHash(runtime.request.data)}` :
+      null;
+  }
+
+  /**
+   * @param {Object} payload
+   * @returns {String}
+   * @private
+   */
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Expected an assignment or function call and instead saw an expression.](https://www.codacy.com/app/MitocGroup/deep-framework/file/1784383532/issues/source?bid=2895475&fileBranchId=3068904#l84)